rules/game_process.tex

```diff
@@ -31,14 +31,20 @@ \subsection{Robot States}
   \item[Unstiff.] It helps to facilitate a consistent and safe handling of the robots for remote competition.
     During any state, if all head buttons are pressed at least one second, the robot should move to a safe seated/crouched position and unstiffen all joints.
     So while in the \texttt{unstiff} state the robot is not allowed to move in any fashion! After booting, the robots are in their \texttt{unstiff} state.
-    Pressing the chest button once while in the \texttt{unstiff} state, permits the robot to stiffen its joints and return to the \texttt{initial} state, or a state as indicated by GameController.
+    Pressing the chest button once while in the \texttt{unstiff} state, permits the robot to stiffen its joints and return to the state indicated by GameController.
 
-  \item[Initial.] The robots are not allowed to be moving in any fashion besides initially standing up.
+    \item[Setup.] The robots are free of any movement at teams convenience.
+    This state is not limited in time and teams have access to the field.
+    The GameController will activate this state before \texttt{initial} (i.e at the beginning of a half and during a timeout).
+
+    \item[Initial.] The robots are not allowed to be moving in any fashion besides initially standing up.
     Shortly pressing the chest button will switch the robot to the \texttt{penalized} state.
+    The robots are preparing to receive a visual signal from the referee.  
```

Comment:
Suggest rewording: In \texttt{iniitial}, the robots are awaiting a visual indication from the referee which signals the transition to \texttt{ready}.